ccxcon/urls.py

```diff
@@ -0,0 +1,18 @@
+from django.conf.urls import include, url
+from django.contrib import admin
+
+from rest_framework_nested import routers
+
+from courses.views import CourseViewSet, ModuleViewSet
+
+router = routers.DefaultRouter()
+router.register('coursexs', CourseViewSet)
+
+modules_router = routers.NestedSimpleRouter(router, 'coursexs', lookup='uuid')
```

Comment:
As it is this doesn't seem to enforce separation between children. I wrote this test:

    def test_separate_children(self):
        course1_dict = {
            "uuid": "39b7ed5e8ace422296101a959fc3359a",
            "title": "title1",
            "author_name": "author1",
            "overview": "overview1",
            "description": "description1",
            "video_url": "http://youtube.com/1",
            "edx_instance": "edx1",
            "price_per_seat_cents": 1,
            "url": "/api/v1/coursexs/39b7ed5e8ace422296101a959fc3359a/"
        }
        resp = self.client.post('/api/v1/coursexs/', course1_dict)
        self.assertEqual(resp.status_code, 201)
        course1_url = json.loads(resp.content.decode('utf-8'))['url']

        course2_dict = {
            "uuid": "39b7ed5e8ace422296101a959fc3359a",
            "title": "title2",
            "author_name": "author1",
            "overview": "overview1",
            "description": "description1",
            "video_url": "http://youtube.com/1",
            "edx_instance": "edx1",
            "price_per_seat_cents": 1,
            "url": "/api/v1/coursexs/39b7ed5e8ace422296101a959fc3359a/"
        }
        resp = self.client.post('/api/v1/coursexs/', course2_dict)
        self.assertEqual(resp.status_code, 201)
        course2_url = json.loads(resp.content.decode('utf-8'))['url']

        module1_dict = {
            "title": "title1",
            "subchapters": "[\"a\", 3]",
            "price_per_seat_cents": 3
        }
        modules1_url = "{url}modules/".format(url=course1_url)
        resp = self.client.post(modules1_url, module1_dict)
        self.assertEqual(resp.status_code, 201)

        module2_dict = {
            "title": "title2",
            "subchapters": "[\"a\", 3]",
            "price_per_seat_cents": 3
        }
        modules2_url = "{url}modules/".format(url=course2_url)
        resp = self.client.post(modules2_url, module2_dict)
        self.assertEqual(resp.status_code, 201)

        resp = self.client.get(modules1_url)
        self.assertEqual(len(json.loads(resp.content.decode('utf-8'))), 1)

It appears they're ending up in the database with the right links, I think they just aren't being queried properly. Maybe the serializer queryset needs filtering?